### PR TITLE
WT-8359 Separate smoke tests from stress tests in Evergreen

### PR DESCRIPTION
**Pull request by lukech**
WT-8321 made changes to let smoke test tasks run the test once (instead of multiple times). Those tasks are no longer suitable to be included in the stress test builders. 

Changes in this PR include:
- Move the smoke test tasks from the stress test builders into the base Ubuntu and Ubuntu ASAN builders. 
- Remove the task tags from smoke test tasks and add the tasks into PPC & zSeries builders explicitly.
- Reduce the times of run to 1 for the smoke test PPC task.

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -2786,34 +2810,6 @@ tasks:
...
-          posix_configure_flags: --enable-diagnostic --with-builtins=lz4,snappy,zlib
-      - func: "format test script"
-        # To emulate the original Jenkins job's test coverage, we are running the smoke test 16 times
-        # Run smoke tests, don't stop at failed tests, use default config
-        vars:
-          test_env_vars:
-            ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0"
-            ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-6.0/bin/llvm-symbolizer
-          format_test_script_args: -S
-          times: 16
```
Comment (lukech) [inline test/evergreen.yml:2816]: This block of code is moved up next to other smoke test tasks. Times reduced to 1, and the task is renamed by removing the "stress" wording

File: test/evergreen.yml [hunk 2]
```diff
@@ -2528,6 +2507,51 @@ tasks:
           # run for 10 minutes.
           format_test_script_args: -t 10 rows=10000 ops=50000
 
+  - name: format-smoke-test
+    exec_timeout_secs: 3600
```
Comment (tammybailey) [inline test/evergreen.yml:2511]: Is this still needed? It's probably worth adding a comment to explain why, if so.

File: test/evergreen.yml [hunk 3]
```diff
@@ -2528,6 +2507,51 @@ tasks:
...
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger with builtins"
+      - func: "format test script"
+        vars:
+          format_test_script_args: -e "SEGFAULT_SIGNALS=all" -b "catchsegv ./t" -S
+
+  - name: format-asan-smoke-test
+    exec_timeout_secs: 3600
```
Comment (tammybailey) [inline test/evergreen.yml:2520]: See above.

**Review by tammybailey — COMMENTED** [review]
Also - we should remove the reference to the smoke tests on lines 651-652.

**Review by keitharnoldsmith — APPROVED** [review]
LGTM!

File: test/evergreen.yml (hunk 2 above)
Comment (lukech) [reply test/evergreen.yml]: Removed this line in 2cb26c5

File: test/evergreen.yml (hunk 3 above)
Comment (lukech) [reply test/evergreen.yml]: Removed this line in 2cb26c5

**Review by tammybailey — APPROVED** [review]
lgtm!

**Pull request merged**